ci/script_linux.sh

```diff
@@ -1,7 +1,7 @@
 make
 if [ $COVERALLS = ON ]; then make coveralls; fi
 sudo ldconfig --verbose # So the test executeables can detect libtinyxml2
-if [ $BUILD_CORE_ONLY = OFF ]; then make test; fi
+if [ $BUILD_CORE_ONLY = OFF ]; then make all test tests; fi
```

Comment:
question: I thought `test` target is to run the tests that are already built. So the order of targets shouldn't be like `make all tests test`? Moreover, I'm not sure why `all` needs to be here. The above `make` wouldn't run for `all` target?